src/BlazorWebFormsComponents/LoginControls/AuthenticateEventArgs.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System;
+using static BlazorWebFormsComponents.LoginControls.Login;
 
 namespace BlazorWebFormsComponents.LoginControls
 {
   public class AuthenticateEventArgs : EventArgs
 	{
-
 		public bool Authenticated { get; set; }
+		public LoginModel loginModel { get; set; }
```

Comment:
I don't like this change, the intend was to simulate the WebForms controls

Reply:
My intention was merely to demonstrate how I would test. Glad you were able to leverage that, and implement your ideal solution :)